docs/docs/Get-Started/get-started-installation.md

```diff
@@ -120,53 +227,53 @@ A confirmation pane containing the selected version's changelog appears.
 3. To change to the selected version, click **Confirm**.
 The application restarts with the new version installed.
 
-## Common installation issues
+## Common OSS installation issues
 
 This is a list of possible issues that you may encounter when installing and running Langflow.
 
-### No `langflow.__main__` module
+* No `langflow.__main__` module
```

Comment:
In the rendered content, the use of bullets for each issue makes the issues hard to pick out among the other text. Suggest using something other than bullets.